From 97d0cb4f4a6f3bec50729e3f896d4a84b796c5c6 Mon Sep 17 00:00:00 2001
From: fidomax <adobegitler@gmail.com>
Date: Tue, 26 Jan 2021 09:17:17 +0300
Subject: [PATCH] Add POSTGIS_DEBUG_LEVEL define to configure.ac

In case of using autoreconf "#define POSTGIS_DEBUG_LEVEL 0" is removed from postgis_config.h.in so build failed with undefined POSTGIS_DEBUG_LEVEL.

Signed-off-by: Maxim Kochetkov <fido_max@inbox.ru>
Fetch from: https://github.com/postgis/postgis/commit/2b2c2cb159cfcaacb75a4e74f17bd2834d61acd0.patch
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index dddb3aef7..86e1597ee 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1131,6 +1131,8 @@ else
     CPPFLAGS="-DNDEBUG $CPPFLAGS"
 fi
 
+AC_DEFINE([POSTGIS_DEBUG_LEVEL], [0], [Define debug level. Default 0])
+
 dnl ===========================================================================
 dnl Allow the developer to disable the automatic updates of postgis_revision.h
 dnl with --without-phony-revision
-- 
2.30.0

